compiler/src/dmd/target.d

```diff
@@ -1141,7 +1325,7 @@ extern (C++) struct Target
      * Returns:
      *  alignment of the stack
      */
-    extern (D) uint stackAlign()
+    extern (C++) uint stackAlign()
```

Comment:
Why this change?

Reply:
It is indeed a copy paste + changing some imports, this might have slipped as this is not an intended change :)